Tools/WebKitTestRunner/InjectedBundle/Bindings/TestRunner.idl

```diff
@@ -58,6 +58,7 @@ interface TestRunner {
     undefined dumpEditingCallbacks();
     undefined dumpSelectionRect();
     undefined dumpTitleChanges();
+    undefined dumpFullScreenOrigin();
```

Comment:
Can this be in testRunnerJS instead of TestRunner.idl?  That's the direction we're heading.